practice-one/src/styles/sections/introduction.css

```diff
@@ -72,7 +72,7 @@
   border-radius: 50%;
   cursor: pointer;
   padding: 9px;
-  left: 424px;
+  left: 90%;
   position: absolute;
```

Comment:
why need absolute position?

Reply:
Because position of icon will base on position of search box ,the icon will stack on search box so we need position absolute to do that. Do you have another idea?